### PR TITLE
feat: item list on new collection page

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] refactor gallery list to item list
- [x] show item list on new collection page
- [x] infinite scroll still not perfect, related to this https://github.com/kodadot/nft-gallery/issues/4952
- [x] closes https://github.com/kodadot/nft-gallery/issues/5023

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Screenshot 2023-02-18 at 13 28 41

### CODE REVIEW

File: components/items/Items.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <ItemsGrid />
+  </div>
+</template>
+
+<script setup lang="ts">
+import SidebarFilter from '@/components/explore/SidebarFilter.vue'
+import ItemsGrid from './ItemsGrid/ItemsGrid.vue'
+</script>
+
+<style scoped></style>
```
Comment (roiLeo) [inline components/items/Items.vue:13]: I don't think it's necessary

File: components/explore/ExploreTabItem.vue [hunk 2]
```diff
@@ -15,26 +15,44 @@
...
-      </NeoButton>
-    </p>
+
+    <!-- TODO: tabs for collection here -->
+    <template v-if="route.name === 'prefix-collection-id'">
+      <p class="control">
+        <NeoButton
+          class="explore-tabs-button"
+          :active="route.name === 'prefix-collection-id'">
+          <span>Items</span>
```
Comment (roiLeo) [inline components/explore/ExploreTabItem.vue:25]:
```suggestion
          <span>{{ $t('items') }}</span>
```

File: components/explore/ExploreTabItem.vue [hunk 3]
```diff
@@ -15,26 +15,44 @@
...
+        </NeoButton>
+      </p>
+      <p class="control">
+        <NeoButton class="explore-tabs-button">
+          <span>Activity</span>
```
Comment (roiLeo) [inline components/explore/ExploreTabItem.vue:30]:
```suggestion
          <span>{{ $t('tabs.activity') }}</span>
```

**Review by roiLeo — COMMENTED** [review]
maybe we should export some composable from `useItemsGrid`, I think we'll need to reuse `useSearchParams` in profile page & collection

File: components/items/Items.vue (hunk 1 above)
Comment (roiLeo) [reply components/items/Items.vue]: (I was referring to components import)

File: components/items/ItemsGrid/ItemsGridImage.vue [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  item.value = {
+    ...props.nft,
+    ...metadata,
+    name: props.nft.name || metadata.name || props.nft.id,
+    image,
+    animation_url,
+    type: metadata.type || '',
+  }
+})
```
[the remark this answers is not in the document]
Comment (vikiival) [reply components/items/ItemsGrid/ItemsGridImage.vue:40]: subsquid

**Review by roiLeo — APPROVED** [review]
looks good

**Comment by exezbcz** [comment]
image
It looks weird on my end. 
otherwise oki

**Comment by exezbcz** [comment]
something is still wrong, is this related to the cards having different height? (afaik there is an issue for this)  
image

link: https://deploy-preview-5039--koda-nuxt.netlify.app/bsx/collection/1299595206?page=2&redesign=true

**Comment by roiLeo** [comment]
> I will check in separate PR 👍 thanks for the link

We already has this issue fix is to wrap card content with `container` class

ref #4935

**Comment by roiLeo** [comment]
Why do we have a full-width layout here compared to explore layout?
We need to unify user experience

**Comment by roiLeo** [comment]
> waiting for this decision actually #4857 should I put wrapper for now?

For now I would just copy same layout as we use currently on explore page.
#4857 fix might come later since we could have potential side effects (didn't test responsive)

**Comment by Jarsen136** [comment]
> Complexity	4

I just wonder if we could manage to reduce it

**Review by Jarsen136 — APPROVED** [review]

File: components/items/ItemsGrid/ItemsGrid.vue [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="is-flex-grow-1">
+    <!-- TODO: breadcrumbs filter here -->
+    <p>total: {{ total }}</p>
```
Comment (daiagi) [inline components/items/ItemsGrid/ItemsGrid.vue:4]: missing translation for 'total'

**Review by daiagi — CHANGES_REQUESTED** [review]

**Comment by exezbcz** [comment]
@preschian apart from this issue I think everything seems to be fine 
https://github.com/kodadot/nft-gallery/pull/5039#issuecomment-1441941344

**Review by daiagi — APPROVED** [review]

**Pull request merged**